ui/components/multichain/account-list-item/account-list-item.stories.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+
+import { Provider } from 'react-redux';
+
+import testData from '../../../../.storybook/test-data';
+import configureStore from '../../../store/store';
+import { AccountListItem } from '.';
+
+const store = configureStore(testData);
+
+const [chaosAddress, simpleAddress, hardwareAddress] = Object.keys(
+  testData.metamask.identities,
+);
+
+const SIMPLE_IDENTITY = {
+  ...testData.metamask.identities[simpleAddress],
+  balance: '0x152387ad22c3f0',
+};
+
+const HARDWARE_IDENTITY = {
+  ...testData.metamask.identities[hardwareAddress],
+  balance: '0x152387ad22c3f0',
+};
+
+const CHAOS_IDENTITY = {
+  ...testData.metamask.identities[chaosAddress],
+  balance: '0x152387ad22c3f0',
+};
+
+const CONTAINER_STYLES = {
+  style: {
+    width: '328px',
+    border: '1px solid var(--color-border-muted)',
+  },
+};
+
+const onClick = () => console.log('Clicked account!');
+
+export default {
+  title: 'Components/Multichain/AccountListItem',
+  component: AccountListItem,
+  argTypes: {
+    identity: {
+      control: 'object',
+    },
+    selected: {
+      control: 'boolean',
+    },
+    onClick: {
+      action: 'onClick',
+    },
+    closeMenu: {
+      action: 'closeMenu',
+    },
+    connectedAvatar: {
+      control: 'text',
+    },
+    connectedAvatarName: {
+      control: 'text',
+    },
+  },
+  args: {
+    identity: SIMPLE_IDENTITY,
+    onClick,
```

Comment:
I think we can remove onClick from args since we already added an action
```suggestion

```

Reply:
Removed in `1bc85b4835af75e68520865018d02932ab4784a3`